src/main/java/core/basesyntax/service/impl/ReportServiceImpl.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.service.ReportService;
+import java.util.Map;
+
+public class ReportServiceImpl implements ReportService {
+    public static final String SEPARATOR = ",";
+
+    @Override
+    public String createReportService() {
+        StringBuilder bd = new StringBuilder("fruit, quantity").append(System.lineSeparator());
+        for (Map.Entry<String, Integer> entry : Storage.FruitStorage.entrySet()) {
+            bd.append(entry.getKey())
+                    .append("SEPARATOR")

```

Comment:
```suggestion
                    .append(SEPARATOR)

```

Did you test your solution and what report you are achieving?)